flox-bash/tests/integration.bats

```diff
@@ -3,10 +3,20 @@
 # flox CLI tests run in two contexts:
 # - unit tests only to be run from within package build
 # - unit and integration tests to be run from command line
-#
+
 
 load test_support.bash
 
+setup_file() {
+  common_setup;
+  if command -v expect >/dev/null 2>&1; then
```

Comment:
Under what circumstances do we not have expect? Is this because Nix expect does not work on Darwin, so we have to rely on the user to install it separately? If so then it might be worth a comment like this:

```suggestion
  # The nixpkgs version of expect does not work on Darwin, so on
  # that system we rely on the user to install expect separately.
  # First discover whether expect is available in the PATH before
  # commencing tests. 
  if command -v expect >/dev/null 2>&1; then
```